functions.php

```diff
@@ -598,6 +636,17 @@ function exhibit_builder_items_search()
     echo $html;
 }
 
+/**
+ * Form for filtering possible cover images.
+ */
+function exhibit_builder_cover_image_filter($exhibit, $formActionUri)
```

Comment:
Is this or its associated partial actually being used? I'm not seeing this function being used anywhere and it doesn't look like the partial's used anywhere else.